src/data/squad.json

```diff
@@ -190,5 +190,83 @@
     "position": "Efficiency Evangelist",
     "image": "/img/pfp/Shade.png",
     "description": "- Amateur Organizational Theorist.\n- Prefers reading over any other activity.\n\nFun fact: Is an actual tree on the internet."
+  },
+  {
+    "name": "Lempire",
+    "position": "Developer",
+    "image": "/img/pfp/Lempire.png",
+    "description": "- Likes to play chess and practice Jiu Jitsu.\n\nFun fact: Likes to read books."
+  },
+  {
+    "name": "Laruku",
+    "position": "QA",
+    "image": "/img/pfp/Laruku.png",
+    "description": "- Background in International Relations.\n- World of Warcraft player for 11 years and counting.\n- (Really bad) Violinist.\n\nFun fact: Buys more stationery than they can use."
+  },
+  {
+    "name": "Funkornaut",
+    "position": "Developer",
+    "image": "/img/pfp/Funkornaut.png",
+    "description": "- Solidity developer.\n- Previously made beer for a living.\n- Big golf guy and yogi.\n\nFun fact: I can wiggle my ears without touching them."
+  },
+  {
+    "name": "Shishigami",
+    "position": "Developer",
+    "image": "/img/pfp/Shishigami.png",
+    "description": "- Designer turned Developer.\n- Obsessed with rendering crazy stuff on displays.\n\n- Fun fact: makes big decisions by talking to plants."
+  },
+  {
+    "name": "Nigiri",
+    "position": "Developer",
+    "image": "/img/pfp/Nigiri.png",
+    "description": "- Recently got Software Engineer degree but don't wanna go through it again.\n- Enjoy playing table tennis on free time.\n- Getting into the entrepeneur world.\n\nFun fact: Was a Taiko player/dancer twice (Japanese drums)."
+  },
+  {
+    "name": "Yaco",
+    "position": "Developer",
+    "image": "/img/pfp/Yaco.png",
+    "description": "- Backend/Data engineer.\n- Enjoy doing stuff that just works.\n\nFun fact: Dulce de batata enjoyer & sudoku puzzle enthusiast."
+  },
+  {
+    "name": "Ilpepepig",
+    "position": "Developer",
+    "image": "/img/pfp/Ilpepepig.png",
```

Comment:
```suggestion
    "image": "/img/pfp/ilpepepig.png",
```

Reply:
I saw this pfp was not loading and its alt text it's "twitter icon", it should be more accurate.

Reply:
https://github.com/defi-wonderland/web/pull/111/commits/cbf49ea9f3d23fd5a7ceb85f993c859078cb4149